ams/ams-api/src/main/java/com/netease/arctic/ams/api/properties/CatalogMetaProperties.java

```diff
@@ -26,6 +26,10 @@ public class CatalogMetaProperties {
     public static final String STORAGE_CONFIGS_KEY_CORE_SITE = "hadoop.core.site";
     public static final String STORAGE_CONFIGS_KEY_HIVE_SITE = "hive.site";
 
+    public static final String STORAGE_CONFIGS_KEY_HDFS_SITE_ID = "hadoop.hdfs.site.id";
```

Comment:
Should we move these properties to moduel AMS ?

Reply:
> Should we move these properties to moduel AMS ?

i remove these all *.id  definition.